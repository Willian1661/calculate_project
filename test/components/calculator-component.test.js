const button = require("../../src/components/button");
const buttonsGroup = require("../../src/components/buttons-group")
const calculatorScreen = require("../../src/components/calculator-component")

describe("calculator screen component", () =>{
    
    const buttons =[ 
        button("foo","fooClass"),
        button("baz","bazClass")
    ]
    it("is passing calculator screen", () => {
        expect(calculatorScreen("body-calc-id","div-screen-id","screen-id",buttonsGroup("buttons-id",buttons))).toMatchSnapshot()
    });

})