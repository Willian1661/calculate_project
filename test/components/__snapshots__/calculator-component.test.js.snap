// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator screen component is passing calculator screen 1`] = `
"
    <div id="body-calc-id">
        <div id="div-screen-id">
            <input type="text" placeholder="0" id="screen-id">
        </div>
            
		<div id="buttons-id">
			<button class="fooClass">foo</button>
			<button class="bazClass">baz</button>}
		</div>
    </div>
    "
`;
